### DISCUSSION TITLE
Compile Test Case FMU

### DISCUSSION THREAD

**Discussion by Norah135** [ibpsa/project1-boptest#669] [Q&A - Test Cases] [answered]
I’m working on the section to compile the Test Case FMU, but the operation appears to be stalled. It takes a long time to compile and ultimately produces no results. Below are the last lines of output I received before the terminal stopped providing any updates. 
<img width="1470" alt="Screenshot 2024-08-07 at 9 32 52 AM" src="https://github.com/user-attachments/assets/de244813-b93e-4379-b8b2-1097de2bfaf6">

In component zon:
  Access to component minT not recommended, it is not present in constraining type of declaration 'replaceable parameter Buildings.ThermalZones.Detailed.Validation.BESTEST.Data.StandardResultsFreeFloating staRes(minT(Min = -18.8+273.15, Max = -15.6+273.15, Mean = -17.6+273.15), maxT(Min = 64.9+273.15, Max = 69.5+273.15, Mean = 66.2+273.15), meanT(Min = 24.2+273.15, Max = 25.9+273.15, Mean = 25.1+273.15)) constrainedby Modelica.Icons.Record'
Warning at line 11, column 7, in file 'BESTESTAir/BaseClasses/Case900FF.mo',
In component zon:
  Access to component maxT not recommended, it is not present in constraining type of declaration 'replaceable parameter Buildings.ThermalZones.Detailed.Validation.BESTEST.Data.StandardResultsFreeFloating staRes(minT(Min = -18.8+273.15, Max = -15.6+273.15, Mean = -17.6+273.15), maxT(Min = 64.9+273.15, Max = 69.5+273.15, Mean = 66.2+273.15), meanT(Min = 24.2+273.15, Max = 25.9+273.15, Mean = 25.1+273.15)) constrainedby Modelica.Icons.Record'
Warning at line 15, column 7, in file 'BESTESTAir/BaseClasses/Case900FF.mo',
In component zon:
  Access to component meanT not recommended, it is not present in constraining type of declaration 'replaceable parameter Buildings.ThermalZones.Detailed.Validation.BESTEST.Data.StandardResultsFreeFloating staRes(minT(Min = -18.8+273.15, Max = -15.6+273.15, Mean = -17.6+273.15), maxT(Min = 64.9+273.15, Max = 69.5+273.15, Mean = 66.2+273.15), meanT(Min = 24.2+273.15, Max = 25.9+273.15, Mean = 25.1+273.15)) constrainedby Modelica.Icons.Record'

**Comment by EttoreZ** [comment, marked as answer]
Dear @Norah135,  from the log you are showing, it is not clear what is wrong. I suggest testing the Modelica model by itself in a different platform (JModelica, OpenModelica, Dymola), and see it works stand alone. Furthermore, BOPTEST test case development is not an easy task, there is a dedicated group within an international project collaboration [Task 3 IBPSA Project 2](https://ibpsa.github.io/project1-boptest/ibpsa/project/meetings/task3/) that deals with test case development. You are more than welcome to attend the monthly meetings. Furthermore, If you think your new test case could be interesting for the community (check the test case application [form](https://docs.google.com/forms/d/e/1FAIpQLSdJpGsaJRflkn1shAos86gPjFEvEMH-gblKPq5tjt8kob8-Pg/viewform?usp=sf_link) you can submit a test case application.
